browser_history/browsers.py

```diff
@@ -125,18 +143,21 @@ class Edge(Browser):
     history_SQL = Chrome.history_SQL
 
 class Opera(Browser):
-    """Opera Browser
```

Comment:
All the browsers have the name on the first line of the comment. Better to have consistency. :smiley: 
Bcoz the documentation is built using these docstrings. Thus, Only this browser might showup a little odd on the HTML page.

Reply:
I had commneted this due to #60 , Ill correct it